### DISCUSSION TITLE
Marker bei Grundstücksuche ausblenden

### DISCUSSION THREAD

**Discussion by nicole-vlbg** [e-netze/webgis-community#132] [Q&A]
Bei uns wurde eine Anfrage gestellt, ob man den Marker bei der Grundstücksuche ausblenden/entfernen kann aber das Grundstück trotzdem highlighten. Der Marker schient wohl in manchen Fällen zu stören.

![image](https://github.com/gis-eni/webgis-forum/assets/124671123/163fa8bc-a4e0-4388-9734-bd9b96cb4a29)

**Comment by gis-eni** [comment]
Das ist leider in WebGIS 5 nicht möglich. 
In WebGIS 6 allerdings, gibt es da in der Tabelle bereits die Funktion "Marker ein/ausblenden"
